src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.db.mixing.properties;
+
+import org.bson.Document;
+import sirius.db.mixing.AccessPath;
+import sirius.db.mixing.EntityDescriptor;
+import sirius.db.mixing.Mixing;
+import sirius.db.mixing.Property;
+import sirius.db.mixing.PropertyFactory;
+import sirius.db.mixing.types.MultiLanguageString;
+import sirius.kernel.commons.Value;
+import sirius.kernel.di.std.Register;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+/**
+ * Represents an {@link MultiLanguageString} field in a {@link sirius.db.mongo.MongoEntity}.
+ * <p>
+ * Multi-Language Strings are stored as a list of nested objects which contain a <tt>lang</tt> and
+ * a <tt>text</tt> properties.
```

Comment:
```suggestion
 * a <tt>text</tt> property.
```

not 100% sure

Reply:
The document has 2 properties: "lang" and "text". But will improve the JavaDoc so they are in the same line